apps/wallet-mobile/src/WalletInit/WalletInitNavigator.tsx

```diff
@@ -38,7 +40,7 @@ export const WalletInitNavigator = () => {
 
       <Stack.Screen
         name="choose-create-restore"
-        component={WalletInitScreen}
+        component={VerifyRecoveryPhrase}
```

Comment:
I think this was added for testing, please don't keep the PR as draft until it is ready.